galsim/bandpass.py

```diff
@@ -93,13 +94,13 @@ def __init__(self, throughput, blue_limit=None, red_limit=None, wave_type='nm',
         # `wave_list` is normally constructed (see `Bandpass.__mul__` below)
 
         self._orig_tp = throughput  # Save this for pickling.
-        self._tp = _tp              # This will normally become orig_tp turned into an actual 
-                                    # function (see _initialize_tp()), although in some cases,
-                                    # it can be supplied directly as a constructor argument.
-
```

Comment:
First example, this comment is eminently more readable than what you changed it too.  Code on the left, comments on the right is a nice paradigm.  So let's ignore the flake8 proscription against hanging indented comments.